Sources/JWTKit/JWTHeaderField.swift

```diff
@@ -0,0 +1,63 @@
+public indirect enum JWTHeaderField: Hashable, Sendable, Codable {
+    case null
```

Comment:
Are there definitely no other types we should support? This adds a new public enum 😛

Reply:
As it happens, you're right, I did (partially) miss one - JSON numbers aren't required to be integers; they can be decimal. JSON considers both integers and decimals to just be numbers, but the most convenient representation in Swift would treat them as separate "types" and add a `case float(Double)` case. (It's a hell of a lot easier than dealing with the 100% representative but bloody painful to handle `NSDecimal` way of doing it)